### PR TITLE
test: [M3-7435, M3-7637] - Cypress tests for OBJ Multicluster access key management

### PR DESCRIPTION
**Pull request by jdamore-linode**
## Description 📝
This adds 3 Cypress tests to confirm access key user flows when the OBJ Multicluster feature is enabled.

I added the `Wait to Merge` label because I'd like to wait for PR #10118 to be merged, at which point I'll push a couple changes here and then will remove the label. I added a couple TODO comments to mark the spots in the tests that will be impacted by the PR.

## Changes  🔄
- Added Cypress tests in `objectStorage/access-keys.smoke.spec.ts`:
  - _can create unlimited access keys with OBJ Multicluster_
  - _can create limited access keys with OBJ Multicluster_
  - _can update access keys with OBJ Multicluster_
- Added a couple intercept utils
- Added a QA data attribute to the selection list component

### Prerequisites
```bash
yarn && yarn build && yarn start:manager:ci
```

### Verification steps 
```bash
yarn cy:run -s "cypress/e2e/core/objectStorage/access-keys.smoke.spec.ts"
```

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by abailly-akamai — APPROVED** [review]
Nice, thank you!

File: packages/manager/cypress/e2e/core/objectStorage/access-keys.smoke.spec.ts [hunk 1]
```diff
@@ -132,4 +146,388 @@ describe('object storage access keys smoke tests', () => {
...
+          ui.actionMenu
+            .findByTitle(
+              `Action menu for Object Storage Key ${mockAccessKey.label}`
+            )
+            .should('be.visible')
+            .click();
+        });
+
+      ui.actionMenuItem
+        // TODO M3-7696 Replace 'Edit Label' with 'Edit' upon merging #10118.
```
Comment (cpathipa) [inline packages/manager/cypress/e2e/core/objectStorage/access-keys.smoke.spec.ts:466]: PR https://github.com/linode/manager/pull/10118 is merged we can update the TODO.

File: packages/manager/cypress/e2e/core/objectStorage/access-keys.smoke.spec.ts [hunk 2]
```diff
@@ -132,4 +146,388 @@ describe('object storage access keys smoke tests', () => {
...
+        .findByTitle('Edit Label')
+        .should('be.visible')
+        .should('be.enabled')
+        .click();
+
+      ui.drawer
+        // TODO M3-7697 Replace 'Edit Access Key Label' with 'Edit Access Key' upon merging #10118.
```
Comment (cpathipa) [inline packages/manager/cypress/e2e/core/objectStorage/access-keys.smoke.spec.ts:473]: Same here

File: packages/manager/cypress/e2e/core/objectStorage/access-keys.smoke.spec.ts [hunk 3]
```diff
@@ -132,4 +146,388 @@ describe('object storage access keys smoke tests', () => {
...
+            .click();
+        });
+
+      ui.actionMenuItem.findByTitle('Permissions').click();
+      ui.drawer
+        .findByTitle(`Permissions for ${mockAccessKey.label}`)
+        .should('be.visible')
+        .within(() => {
+          mockBuckets.forEach((mockBucket) => {
+            // TODO M3-7733 Update this selector when ARIA label is fixed.
```
Comment (cpathipa) [inline packages/manager/cypress/e2e/core/objectStorage/access-keys.smoke.spec.ts:395]: Thank you @jdamore-linode for creating the ticket.

**Review by cpathipa — APPROVED** [review]
Nice work @jdamore-linode

**Comment by jdamore-linode** [comment]
Thanks @cpathipa! Just merged in `develop` and updated the tests accordingly -- if everything passes I'll plan to merge 👍

**Pull request merged**